src/extension/background-script/actions/webbtc/index.ts

```diff
@@ -0,0 +1,6 @@
+import getAddress from "~/extension/background-script/actions/webbtc/getAddress";
+import getAddressWithPrompt from "~/extension/background-script/actions/webbtc/getAddressWithPrompt";
```

Comment:
why is this using a different import path than with `./getinfo`?

Reply:
I made them all relative for now. I think actually it would be good to do the opposite though and use an eslint rule to have consistent absolute imports, and add extra aliases in tsconfig so these imports aren't so long. Maybe "@background-script/actions/webbtc/getAddressWithPrompt"

Reply:
I will create a separate issue for this